test/miner_poc_SUITE.erl

```diff
@@ -670,7 +670,7 @@ setup_dist_test(TestCase, Config, VarMap) ->
     true = load_genesis_block(GenesisBlock, Miners, Config),
     miner_fake_radio_backplane ! go,
     %% wait till height 50
-    true = wait_until_height(Miners, 50),
+    true = wait_until_height(Miners, 10),
```

Comment:
maybe we should also lower the `poc_challenge_interval` for testing from 20 -> 10?

Reply:
Tried this, it breaks the test